ref_builder/otu/utils.py

```diff
@@ -139,7 +139,7 @@ def _get_segments_from_records(records: list[NCBIGenbank]) -> list[Segment]:
             segments.append(
                 Segment(
                     id=segment_id,
-                    name=record.source.segment,
+                    name=str(get_multipartite_segment_name(record)),
```

Comment:
I think we may as well change the type of this field to `SegmentName` instead of converting it to a string.

Reply:
Still have to deal with all the monopartite stuff. Should we set the SegmentName.key to an empty string in that case?

Reply:
Closing, will handle in an upcoming PR.